components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -5060,7 +5060,9 @@ private ServiceProviderProperty buildIsAPIBasedAuthenticationEnabledProperty(Ser
         ServiceProviderProperty androidPackageName = new ServiceProviderProperty();
         androidPackageName.setName(ANDROID_PACKAGE_NAME_PROPERTY_NAME);
         androidPackageName.setDisplayName(ANDROID_PACKAGE_NAME_DISPLAY_NAME);
-        androidPackageName.setValue(String.valueOf(clientAttestationMetaData.getAndroidPackageName()));
+        if (clientAttestationMetaData.getAndroidPackageName() != null) {
```

Comment:
instead of a null check shall we use StringUtils.isNotBlank? in case from the ui if an empty string is passed.

Reply:
Fixed with cd5db0a89aaaa5616efc836d1975e821318dd403